dateparser/conf.py

```diff
@@ -129,6 +133,28 @@ def _check_parsers(setting_name, setting_value):
     _check_repeated_values(setting_name, setting_value)
 
 
+def _check_default_languages(setting_name, setting_value):
+    is_valid = True if map_languages(setting_value) else False
+    if not is_valid:
+        raise SettingValidationError(
+            'Given list is not a valid value for {}. It can take languages supported by'
+            ' dateparser.'.format(
+                setting_name,
+            )
+        )
```

Comment:
I think you shouldn't use map_languages here. What you can do is just test if the values are not valid languages, and maybe you can also check if there are repeated values, like in `_check_parsers()` :)

```suggestion
def _check_default_languages(setting_name, setting_value):
    unsupported_languages = set(setting_value) - set(language_order)
    if unsupported_languages:
        raise SettingValidationError(
            'Found invalid languages in the "{}" setting: {}'.format(
                setting_name, ', '.join(map(repr, unsupported_languages))
            )
        )
    _check_repeated_values(setting_name, setting_value)
```

Reply:
Done 👍🏻